doc/2/api/controllers/auth/create-api-key/index.md

```diff
@@ -0,0 +1,103 @@
+---
+code: true
+type: page
+title: createApiKey
+---
+
+# createApiKey
+
+Creates a new API key.
```

Comment:
Yes I should not allow anonymous users to manage API keys.  

Concerning `auth:` methods, I agree that we didn't have an usecase for now but this is something very common to allow individual users to generate their API keys (eg: Github).  

IMHO, we can expose these methods since not a lot of specific code is involved, what do you think?

Reply:
Ok, github is actually a fine example of why we would want to be able to create our own API key. 
Thinking about it, it could also be useful for IoT devices going through an auto-provisionning procedure.